models/repo/search.go

```diff
@@ -8,14 +8,14 @@ import "code.gitea.io/gitea/models/db"
 // SearchOrderByMap represents all possible search order
 var SearchOrderByMap = map[string]map[string]db.SearchOrderBy{
 	"asc": {
-		"alpha":   db.SearchOrderByAlphabetically,
+		"alpha":   "owner_name ASC, name ASC",
```

Comment:
also fix alpa sort in explore

Reply:
So this change will affect not only dashboard but also other places? Could you list all the affected places in this PR.

Reply:
Yep, agreed, it does seem to be a bit over-eager:
In many cases, you only want to search within your own repos.
Thinking about it…
If that's the case, all repos have the same ownername, so the first part is simply ignored.
Okay, fine then.